### ISSUE TITLE
Stitching seams

### ISSUE THREAD

**Issue by sjlynch** [mapbox/martini#7]
Cool library! I tiled a bunch of RTIN terrains together and there is a seams issue that I was unable to resolve. The edges of each mesh has an irregular number of vertices, so it is not sufficient to average each edge's height because the vertices don't line up. Adding difference thresholds (i.e. average each vertex as long as it is within some lat/lng range) will work but it results in texture stretching. Is there any plan to address seams? Included some screenshots below. I'm on a gtx 1080 getting 120fps with this many tiles. 

Another method, directly applying height to a planebuffergeometry, works and is easy to stitch but I get 55fps with that method. Clearly RTINs offer an immense amount of optimization, but I don't know where to go from here with the seams issue. I could try hiding seams with skirts? Is there a better way?

![image](https://user-images.githubusercontent.com/5000496/68341467-809c5b80-00b6-11ea-9843-abeff187651f.png)

![image](https://user-images.githubusercontent.com/5000496/68341439-72e6d600-00b6-11ea-8661-fdba52448332.png)

**Comment by mourner** [comment]
Yeah, I think skirts are the easiest option — probably worth adding an API for generating them to the library.

**Comment by mourner** [comment]
Equivalent issue in Delatin: https://github.com/mapbox/delatin/issues/2

**Comment by chrisgervang** [comment]
How could I generate skirts?

**Comment by kylebarron** [comment]
Making skirts inside Martini itself would improve performance compared to implementing it downstream so that the `triangles` array wouldn't have to be copied into a new, larger array.

It looks like `getMesh` would have to be modified with an optional `skirt` argument, and then both the initial scan to count the required number of vertices and triangles and the second pass filling them into the arrays would need to be updated. I'm still struggling to actually wrap my head around the code, so I'm not sure I'll be able to write a PR for this.

**Comment by ezybaker** [comment]
For tile stitching, this obviously wouldn't work if you're going on a tile-by-tile basis, but if you already have all the elevation tiles beforehand, you can do 2 passes for the errors. The first pass can just be the createTile that runs update() and creates the errors for every tile, and the second pass simply runs a slightly changed update function that takes the max error between the current tile and the neighboring tile for all the edges. From my limited testing and unoptimized code it does add maybe 25% increased time or so but the tradeoff is there if desired and it's not hard to implement.
